### DISCUSSION TITLE
Disable updating casks

### DISCUSSION THREAD

**Discussion by cyberbob** [Homebrew/discussions#3518] [Casks] [answered]
Hello,
I'm using Brewfile and have cask line for all my apps. Because I'm using MacUpdator I don't need to upgrade those cask'ed Applications during brew upgrade. Some applications won't be updated because of auto_updates true flag, but some of them will. Is there way to exclude casks from brew upgrade process ?

**Comment by SMillerDev** [comment, marked as answer]
Unless they auto update, no.

**Comment by cyberbob** [comment]
Thank You. I'll create issue with request for such functionality.

**Reply by SMillerDev** [reply]
Homebrew only accepts feature requests as pull requests. But if you make one of those I'll happily review it.
